problems/reverseString.yaml

```diff
@@ -0,0 +1,26 @@
+id: 4
+title: ReverseString
+text: |
+  Write a function called reverseString that takes a string parameter 
+  and returns the reverse version of said string.
+  
+  Refer to example below.
+  
+  reverseString(YellowBrickCode) = edoCkcirBwolleY
+  reverseString(Google) = elgooG
+code: |
+  function reverseString(str) {
+    return str;
+  }
+tests:
+  - assert(reverseString('hello') === 'olleh');
+  - assert(reverseString('Hello World') === 'dlroW olleH');
+  - assert(reverseString('YellowBrickCode') === 'edoCkcirBwolleY');
+solution: |
+    function reverseString(str) {
+        var newString = "";
+        for (var i = str.length - 1; i >= 0; i--) {
```

Comment:
Since we are trying to teach writing elegant code, maybe we should emphasize usage of existing functionalities in our solutions too.

For example, the solution to this problem can be:
```
function reverseString(str) {
  return str.split('').reverse().join('');
}
```
Although we are not displaying our solution currently, this will make us better prepared if we ever want to do so.